<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>657</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>620</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>12560</width>
    <height>6220</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CluVis - Wizualizator Skupień</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Wizualizacja</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QStackedWidget" name="stackedWidgetSettings">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>303</width>
              <height>540</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>303</width>
              <height>540</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="pageGeneral">
             <widget class="QGroupBox" name="groupBoxSettingsGeneral">
              <property name="geometry">
               <rect>
                <x>9</x>
                <y>9</y>
                <width>285</width>
                <height>530</height>
               </rect>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>285</width>
                <height>530</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>241</width>
                <height>530</height>
               </size>
              </property>
              <property name="title">
               <string>Ustawienia ogólne</string>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QLabel" name="labelObjectType">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Typ grupowanych obiektów:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelObjectTypeValue">
                 <property name="text">
                  <string>&lt;b&gt;RSES LEM2 Rules&lt;/b&gt;</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelAlgorithmGrouping">
                 <property name="text">
                  <string>Algorytm grupowania:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelAlgorithmGroupingValue">
                 <property name="text">
                  <string>&lt;b&gt;Klasyczny hierarchiczny algorytm grupowania&lt;/b&gt;</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelObjectBase">
                 <property name="text">
                  <string>Baza reguł:</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_6">
                 <item>
                  <widget class="QLabel" name="labelObjectBaseName">
                   <property name="text">
                    <string>&lt;b&gt;Nie wczytano.&lt;/b&gt;</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="labelIsBaseGrouped">
                   <property name="text">
                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;(Niepogrupowana) &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QLabel" name="labelObjectsNumber">
                 <property name="text">
                  <string>Liczba reguł w bazie:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelObjectsNumberValue">
                 <property name="text">
                  <string>&lt;b&gt;0.&lt;/b&gt;</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelRuleGroupingMethod">
                 <property name="text">
                  <string>Grupowanie reguł ze względu na:</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <widget class="QComboBox" name="comboBoxRuleGroupedPart">
                   <item>
                    <property name="text">
                     <string>Warunki</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Konkluzje</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_2">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeType">
                    <enum>QSizePolicy::Fixed</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>100</width>
                     <height>28</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Miara odległości między obiektami:</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QComboBox" name="comboBoxInterobjectDistanceMeasure">
                   <property name="minimumSize">
                    <size>
                     <width>157</width>
                     <height>0</height>
                    </size>
                   </property>
                   <item>
                    <property name="text">
                     <string>Miara Gowera</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Simple Similarity</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Weighted Similarity</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_5">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QLabel" name="labelInterclusterDistanceMeasure">
                 <property name="text">
                  <string>Miara odległości międzyklastrowej:</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayoutInterclusterDistanceMeasure">
                 <item>
                  <widget class="QComboBox" name="comboBoxInterclusterDistanceMeasure">
                   <item>
                    <property name="text">
                     <string>Pojedynczego połączenia</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Całkowitego połączenia</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Średniej grupowej</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Środków ciężkości</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeType">
                    <enum>QSizePolicy::Fixed</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>100</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QLabel" name="labelStopCondition">
                 <property name="text">
                  <string>Na ilu skupieniach zakończyć grupowanie?
Liczba skupień musi być mniejsza lub równa
liczbie obiektów w bazie.</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayoutStopCondition">
                 <item>
                  <widget class="QSpinBox" name="spinBoxStopConditionValue">
                   <property name="minimum">
                    <number>1</number>
                   </property>
                   <property name="maximum">
                    <number>16777215</number>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_3">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeType">
                    <enum>QSizePolicy::Fixed</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>100</width>
                     <height>28</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QLabel" name="labelAlgorithmVisualization">
                 <property name="text">
                  <string>Algorytm wizualizacji:</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayoutAlgorithmVisualization">
                 <item>
                  <widget class="QComboBox" name="comboBoxAlgorithmVisualization">
                   <item>
                    <property name="text">
                     <string>RT Slice And Dice</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Circular Treemap</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_4">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeType">
                    <enum>QSizePolicy::Fixed</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>100</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QCheckBox" name="checkBoxVisualizeAllHierarchyLevels">
                 <property name="text">
                  <string>Wizualizuj wszystkie poziomy hierarchii.</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="checkBoxSearchForBestIndexes">
                 <property name="text">
                  <string>Szukaj optymalnych grupowań.</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayoutButtons">
                 <item>
                  <widget class="QPushButton" name="pushButtonGroup">
                   <property name="text">
                    <string>Grupuj</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButtonVisualize">
                   <property name="text">
                    <string>Wizualizuj</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </widget>
            <widget class="QWidget" name="pageDetails">
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QGroupBox" name="groupBoxSpecificSettings">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>285</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>241</width>
                  <height>158</height>
                 </size>
                </property>
                <property name="title">
                 <string>Ustawienia szczegółowe</string>
                </property>
                <property name="flat">
                 <bool>false</bool>
                </property>
                <property name="checkable">
                 <bool>false</bool>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_3"/>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>30</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGraphicsView" name="graphicsView">
          <property name="minimumSize">
           <size>
            <width>102</width>
            <height>60</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>10260</width>
            <height>6020</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Dziennik zdarzeń</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QTextBrowser" name="textBrowserLog">
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Plik</string>
    </property>
    <addaction name="actionLoadObjectBase"/>
    <addaction name="separator"/>
    <addaction name="actionSaveVisualization"/>
    <addaction name="actionGenerateReport"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Pomoc</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuWidok">
    <property name="title">
     <string>Widok</string>
    </property>
    <widget class="QMenu" name="menuJ_zyk">
     <property name="title">
      <string>Język</string>
     </property>
     <addaction name="actionAngielski"/>
     <addaction name="actionPolski"/>
    </widget>
    <addaction name="menuJ_zyk"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuWidok"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionLoadObjectBase">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Wczytaj bazę obiektów</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Zakończ program</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>Informacje o programie</string>
   </property>
  </action>
  <action name="actionSaveVisualization">
   <property name="text">
    <string>Zapisz wizualizację</string>
   </property>
  </action>
  <action name="actionGenerateReport">
   <property name="text">
    <string>Generuj raport</string>
   </property>
  </action>
  <action name="actionAngielski">
   <property name="text">
    <string>Angielski</string>
   </property>
  </action>
  <action name="actionPolski">
   <property name="text">
    <string>Polski</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
